HeimdallrTests/Core/HeimdallrSpec.swift

```diff
@@ -725,6 +725,34 @@ class HeimdallrSpec: QuickSpec {
                         expect(result?.error?.code).to(equal(OAuthErrorInvalidClient))
                     }
                 }
+
+                context("when issueing multiple requests at the same time") {
+                    it("only issues one refresh token call") {
```

Comment:
Unfortunately, the second request does not belong to the second call of `authenticateRequest`, but also to the first one.
When everything works as expected, the second call of `authenticateRequest` does not make any network requests.

And that property (only making requests when things are not working) is what I based the current test on.